src/tools.cpp

```diff
@@ -1162,3 +1165,30 @@ const char* getReturnMessage(ReturnValue value)
 			return "Sorry, not possible.";
 	}
 }
+
+bool loadDirectory(const std::string directory)
```

Comment:
Function name should tell us what does the function do.
You've moved it outside luascript.cpp, so now it could be hard to tell how the function works without checking its definition.